Exclude log_outputs() from execute profiling scope

tarun292 opened the pull request:
Summary: `log_outputs()` should not be included in the `execute()` profiling scope as it distorts the profiling numbers especially on smaller embedded platforms like Xtensa.

Differential Revision: D64541740

mcremon-meta reviewed (approved):
LGTM when linting is good! Thanks  :)

tarun292 committed:
Exclude log_outputs() from execute profiling scope (#6325)

Summary:

`log_outputs()` should not be included in the `execute()` profiling scope as it distorts the profiling numbers especially on smaller embedded platforms like Xtensa.

Reviewed By: mcremon-meta

Differential Revision: D64541740
